apps/jubmoji-quest/prisma/schema.prisma

```diff
@@ -51,18 +58,24 @@ model Power {
   sigNullifierRandomness String? // used if we want unique nullifiers for each power
   powerType              PowerType
   powerParams            Json    // params for the specific power type
+  proofType              ProofType?
+  proofParams            Json?    // params for the specific proof type
+  createdAt              DateTime @default(now())
 
   // Relationships
-  questId       Int  
-  quest         Quest   @relation(fields: [questId], references: [id])
-  qrCodes       QRCode[]   
-  sigNullifiers PowerSigNullifier[]
+  questId                   Int  
+  quest                     Quest   @relation(fields: [questId], references: [id])
+  prerequisiteCards         Card[]  @relation("PowerPrerequisiteCards") // cards needed to see the power
```

Comment:
Shouldn't these two relations be optional? for the exitsing powers that don't have these

Reply:
there is no Card[]? type